### DISCUSSION TITLE
Unable to find the ASO-S satellite in the software

### DISCUSSION THREAD

**Discussion by 1234f503** [Stellarium/stellarium#3425] [Q&A] [answered]
Unable to find the ASO-S satellite in the software，Advanced Space-based Solar Observatory，This is its name.This is its launch date.08.10.2022.Can't find this satellite?

**Comment by alex-w** [comment, marked as answer]
![stellarium-001](https://github.com/Stellarium/stellarium/assets/88731/96212aa3-fe7d-490c-95ac-6f112185c24b)

You need to enable "active" group of TLE's

**Reply by 1234f503** [reply]
I finally found it
